Userland/Utilities/icc.cpp

```diff
@@ -48,8 +55,12 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     if (auto color_management_module_bits = flags.color_management_module_bits())
         outln("                            CMM bits: 0x{:04x}", color_management_module_bits);
 
-    out_optional("   device manufacturer", profile->device_manufacturer());
-    out_optional("          device model", profile->device_model());
+    out_optional("   device manufacturer", TRY(profile->device_manufacturer().map([](auto device_manufacturer) {
+        return hyperlink(device_manufacturer_url(device_manufacturer), device_manufacturer);
+    })));
+    out_optional("          device model", TRY(profile->device_model().map(([](auto device_model) {
+        return hyperlink(device_model_url(device_model), device_model);
+    }))));
```

Comment:
super nit: It looks like you have one extra set of brackets here:
```suggestion
    out_optional("          device model", TRY(profile->device_model().map([](auto device_model) {
        return hyperlink(device_model_url(device_model), device_model);
    })));
```

Reply:
Thanks!